src/Template/Element/advanced_search_form.ctp

```diff
@@ -29,6 +29,21 @@ echo $this->Form->create('AdvancedSearch', [
 ]);
 ?>
 
+<?php if ($this->request->getParam("?")): ?>
+    <md-divider>
+        <md-button class="md-primary md-raised"
+                    href="<?= $this->Url->build([
+                        "controller" => "Sentences",
+                        "action" => "advanced_search",
+                        "?" => [],
+                    ]);?>">
+            <?= __('Clear') ?>
+        </md-button>
+        <i><?= __("This form has been filled by a template.")?></i>
+    </md-divider>
```

Comment:
I thought the `<md-divider>` tag wasn’t supposed to contain anything. :thinking: